chapters/installation.tex

```diff
@@ -54,11 +54,11 @@ \section{Installing on macOS}
 
 \section{Installing on Linux}
 
-For Linux, you can download the latest Python 3 binary releases from \url{https://www.python.org/downloads/linux/} or use your package manager to install it. To make sure you have Python 3 installed on your system, run $python3$ in your terminal.
+To install python on Linux, you can download the latest Python 3 binary releases from \url{https://www.python.org/downloads/source/} or use your package manager (apt-get, aptitude, synaptic and others) to install it. To make sure you have Python 3 installed on your system, run $python3 --version$ in your terminal.
```

Comment:
As far I see there are only sources here: https://www.python.org/downloads/source/ - not 
"binary releases".

Reply:
Yes, the url should be the same as before, and python should be Python (uppercased). Also, if we do --version it won't match anymore the screenshot of the repl and I want people to know how to enter the python repl by that time..

Reply:
Url https://www.python.org/downloads/linux/ throws 404... So I think it is okay to change url What I am really referring that there are not binary downloads - just sources.

Reply:
Yes, the old url is not working anymore.

I moved the screenshot of REPL under to last instruction that you also recommend running python3 to open REPL. I think is more intuitive use 2 different commands in this case because the user can learn about --version command and he will learn how to use REPL in the briefly [next line](https://github.com/filipefilardi/full-speed-python/blob/master/chapters/installation.tex#L59).